Sources/AWSLambdaEvents/AppSync.swift

```diff
@@ -146,7 +146,7 @@ public struct AppSyncEvent: Decodable {
     }
 }
 
-public enum AppSyncResponse<ResultType: Encodable>: Encodable {
+public enum AppSyncResponse<ResultType: Encodable & Sendable>: Encodable, Sendable {
```

Comment:
You are adding a sendable requirement to `ResultType`. Again this should only be Sendable when `ResultType` is Sendable. You will need to do this as an extension
```swift
extension AppSyncResponse: Sendable where ResultType: Sendable {}
```

Reply:
OK done